src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServer.cs

```diff
@@ -63,7 +63,7 @@ public static Task<RazorLanguageServer> CreateAsync(Stream input, Stream output,
             Serializer.Instance.JsonSerializer.Converters.RegisterRazorConverters();
 
             // Custom ClientCapabilities deserializer to extract experimental capabilities
-            Serializer.Instance.JsonSerializer.Converters.Add(ExtendableClientCapabilitiesJsonConverter.Instance);
+            // Serializer.Instance.JsonSerializer.Converters.Add(ExtendableClientCapabilitiesJsonConverter.Instance);
```

Comment:
Will either remove/fix in the main code actions PR. Just committing this right now to ensure we aren't in a broken state.